File: Beginner/1005.java
import java.util.Scanner;
import java.util.Locale;

class Main {
	public static void main(String[] args){
		Scanner scan = new Scanner(System.in).useLocale(Locale.US);
		Locale.setDefault(Locale.US);

		double a = scan.nextDouble();
		double b = scan.nextDouble();
		scan.close();

		double average = ((a * 3.5) + (b * 7.5)) / 11.0;

		System.out.printf("MEDIA = %.5f\n", average);
	}
}

File: Beginner/1005.c
#include <stdio.h>

int main (){
	double number1, number2, average; 
	scanf("%lf", &number1);
	scanf("%lf", &number2);
	average = ((number1 * 3.5) + (number2 * 7.5)) / 11;
	printf("MEDIA = %.5f\n", average);
	return 0; 
}
